hw_diag/templates/diagnostics_page.html

```diff
@@ -28,7 +28,13 @@ <h3>{{ diagnostics.error }}</h3>
         <div class="text-center pb-4 mb-4 border-bottom border-light">
             <h1>Diagnostics Information</h1>
             <br />
-            <h3>{{ diagnostics.AN }}</h3>
+            <h3>
+              {% if diagnostics.AN == "ECC failure" %}
```

Comment:
Is there existing logic that assumes `AN` will be `ECC failure`? If not, can we use `None` instead? That would be more consistent with how we are handling Helium API failures:

Reply:
It seems like we are setting it to ECC failure [here](https://github.com/NebraLtd/hm-diag/blob/c22a42915cb4f029dae41f31076d9867f84894dd/hw_diag/utilities/hardware.py#L255)

I can change it to be ```None``` here I reviewed the rest of the references to public keys in hm_diag and it seems like this is the only spot we provide a different result than the original. Then in hm_pyhelper it seems like it solely relies on gateway_mfr to return the public keys result